gensim/models/phrases.py

```diff
@@ -391,15 +391,14 @@ def load(cls, *args, **kwargs):
                     raise ValueError(f'failed to load {cls.__name__} model, unknown scoring "{model.scoring}"')
 
         # common_terms didn't exist pre-3.?, and was renamed to connector in 4.0.0.
-        if hasattr(model, "common_terms"):
-            model.connector_words = model.common_terms
-            del model.common_terms
-        else:
-            logger.warning(
-                'older version of %s loaded without common_terms attribute, setting connector_words to an empty set',
-                cls.__name__,
-            )
-            model.connector_words = frozenset()
+        if not hasattr(model, "connector_words"):
+            if hasattr(model, "common_terms"):
+                model.connector_words = model.common_terms
+                del model.common_terms
+            else:
+                logger.warning('older version of %s loaded without common_terms attribute', cls.__name__)
+                logger.warning('setting connector_words to an empty set')
```

Comment:
Please keep the warning as one line.

Reply:
One line warning was making the line 121 characters long and a flake test was failing, so I had to keep it in into two lines warning like all other already existing warnings e.g lines 404 and 405

Reply:
I think what @piskvorky means here is that you should output a single _warning_ only. You're outputting two warnings here, where one would do (the warning can occupy multiple lines in code, as in the original code).

Reply:
Yes, that's what I meant, thanks. If other parts of the code do it badly (split a single warning message into multiple isolated warnings), feel free to fix them too :)

Reply:
@aloknayak29 Please have a look at this.

Reply:
Yes, converted it to a single line warning